### PR TITLE
added button component.

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

**Comment by DivyanshSareen** [comment]
image

you can align the text with the icon here

File: css/common.css [hunk 1]
```diff
@@ -142,7 +145,7 @@ h2{
 }
 .nav-brand{
     text-align:left;
-    margin:2rem 0 2rem 2rem;
+    margin:1rem 0 1rem 4.5rem;
 }
 .navigation-container .link{
     /* background-color: #56cfe1; */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code.

File: css/common.css [hunk 2]
```diff
@@ -158,15 +161,18 @@ h2{
 /* side-navigation */
 .side-navigation-container{
     background-color: var(--background-color);
-    font-size:1.2rem;
+    /* font-size:1.2rem; */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code.

File: css/common.css [hunk 3]
```diff
@@ -158,15 +161,18 @@ h2{
...
+    font-size:1rem;
     height: 100%;
     left: 1.87rem; 
     overflow-x: hidden;
     overflow-y: auto;
-    padding-top:4rem;
+    /* padding-top:4rem; */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code.

File: css/common.css [hunk 4]
```diff
@@ -158,15 +161,18 @@ h2{
...
+    padding-top:0;
     position: fixed;
     top: 12.5rem;
-    width: 200px;
+    /* width: 200px; */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code.

File: css/common.css [hunk 5]
```diff
@@ -182,7 +188,8 @@ ul{
 .nav-content {
     color: var(--alert-secondary-color);
     display: block;
-    font-size: 1.8rem;
+    /* font-size: 1.8rem; */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]
Do the suggested changes

**Review by dev-prabhat — APPROVED** [review]
Looks good to me

File: components/button/button.css [hunk 6]
```diff
@@ -78,10 +77,16 @@
   .btn-icons{
     background-color: #4361ee;
     color:white;
+    font-size: 2.1rem;
     height:5rem;
     width:14rem;
     text-align:left;
   }
+  .fa-2x {
+    font-size: 30px;
```
Comment (zoef-yt) [inline components/button/button.css]: Use rem

File: components/button/button.css [hunk 7]
```diff
@@ -78,10 +77,16 @@
...
+    vertical-align: middle;
+    margin-right: 5px;
+  }
   
   /* floating action btn */
   .btn-floating-action{
```
Comment (zoef-yt) [inline components/button/button.css:92]: use variable in the below code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: css/common.css [hunk 8]
```diff
@@ -16,29 +16,51 @@
...
 
 /* landing page */
 .left-header{
-    color:black;
+    color:var(--primary-color);
     font-size:3rem;
     font-weight: 700;
     margin:4rem;
     padding-top:10rem;
+    /* margin:1rem auto 1rem 4.8rem; */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code

File: css/common.css [hunk 9]
```diff
@@ -16,29 +16,51 @@
...
     text-align: right;
 }
 .left-nav{
     text-align: left;
+    margin:1rem auto 0 4.8rem;
 }
 .right-nav li{
     font-weight: 500;
-    margin: 0 1rem;
+    /* margin: 0 1rem; */
```
Comment (zoef-yt) [inline css/common.css]: remove commented code

File: css/common.css [hunk 10]
```diff
@@ -16,29 +16,51 @@
...
     list-style: none;
     display: inline;
-    margin-block-start: 0;
+    padding: 20px 10px;
```
Comment (zoef-yt) [inline css/common.css]: use rem

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Good code. Good to megre?

**Comment by jDineshKarthic** [comment]
The items (icon and the text) and not aligned properly in the same line. Try adding 
display :flex;
align-items: center; for all the button tags. 
And add display: flex , to the container div tag.
button

**Pull request merged**